zone/effects.cpp

```diff
@@ -1096,7 +1096,7 @@ void EntityList::AESpell(
 		max_targets = nullptr;
 	}
 
-	int max_targets_allowed = RuleI(Range, AOEMaxTargets); // unlimited
+	int max_targets_allowed = 4;
```

Comment:
Why is this fixed now ?

Reply:
This is the default for Mesmerize and Lull spells. Whereas before we were doing this in reverse, which could have caused issues if the rule actually functioned.